tests/iio_stresstest.c

```diff
@@ -82,11 +89,14 @@ static size_t cache_line_size(void)
 
 #elif __linux__
 	FILE * p = 0;
+	int ret;
 
 	p = fopen("/sys/devices/system/cpu/cpu0/cache/index0/coherency_line_size", "r");
 	if (p) {
-		fscanf(p, "%zu", &cacheline);
+		ret = fscanf(p, "%zu", &cacheline);
 		fclose(p);
+		if (!ret || ret == EOF)
```

Comment:
maybe coverity would complain that negative values are checked?
no idea;

in any case, maybe let's go for the simplest check for `ret = fscanf()`; i.e. `if (ret != 1) cacheline = 0`;
we know that the function should return exactly 1 scanned element;
enforcing anything-but 1 scanned elem returned should cover all other returns;

Reply:
updated